### PR TITLE
[Hackday] [Toolbar] Fix info position and icons on small screens

### PR DESCRIPTION
**Pull request by stefanosala**
| Q | A |
| --- | --- |
| Bug fix? | yes |
| New feature? | no |
| BC breaks? | no |
| Deprecations? | no |
| Tests pass? | yes |
| Fixed tickets | #12582 |
| License | MIT |
| Doc PR | - |

I changed a bit the order of profiles, IMO the dump goes closer to request data. Then I tweaked a bit the toolbar to show more icons even on smaller screen. I tested everything with Chrome and IE9 and it seems to work smoothly.

Here are some explanatory screenshots:

### CODE REVIEW

File: src/Symfony/Bundle/DebugBundle/Resources/config/services.xml [hunk 1]
```diff
@@ -11,7 +11,7 @@
         </service>
 
         <service id="data_collector.dump" class="Symfony\Component\HttpKernel\DataCollector\DumpDataCollector">
-            <tag name="data_collector" id="dump" template="@Debug/Profiler/dump.html.twig" />
+            <tag name="data_collector" id="dump" template="@Debug/Profiler/dump.html.twig" priority="285" />
```
Comment (fabpot) [inline src/Symfony/Bundle/DebugBundle/Resources/config/services.xml]: Can you revert the priority changes?

**Comment by stefanosala** [comment]
@fabpot done!

**Comment by stof** [comment]
According to your sample app, there is some issues where some panels are getting smaller width than what they should on hover, making it impossible to read the content (look at the dump panel for instance)

**Comment by stof** [comment]
however, allowing icons to go on several lines when there is not enough space on 1 line to display the whole toolbar is a simple way to fix the issue of items disappearing

**Comment by stefanosala** [comment]
@stof thanks, it wasn't easy, but I spotted the bug :)

**Comment by stof** [comment]
:+1:

**Comment by fabpot** [comment]
Thank you @stefanosala.

**Pull request merged**